### DISCUSSION TITLE
Mapping Prometheus into Datadog metrics

### DISCUSSION THREAD

**Discussion by szibis** [vectordotdev/vector#18734] [General]
Hi Team. Based on this https://docs.datadoghq.com/integrations/guide/prometheus-metrics/?tab=latestversion all metrics coming into Vector from any prometheus source whatever is scrape or remote_write metrics should be mapped probably in Datadog Metrics sink to achieve same results as in data coming for example from Datadog-Agent. Is there any solution planed do such thing?
Probably any format supporting count, gauge, histogram or distribution like metrics should be at the end converted into format that will use all Datadog features and not be only 1:1 metrics store from source to land in DD which is not what we expect here. This also may be done in aggregation transform or something similar.

**Comment by jszwedko** [comment]
Hi @szibis !

I believe this is already done. Are you finding a particular metric type that is not converted correctly?
